_data/sites.json

```
@@ -2736,6 +2736,16 @@
         ]
     },
 
+    {
+        "name": "Chick-fil-A",
+        "url": "https://privacyportal.onetrust.com/webform/63dc78c7-5612-4181-beae-47dead0569ee/01252c81-82df-4db5-9cb8-bf23a62a9a55",
+        "difficulty": "hard",
+        "notes": "Fill out the provided <a href='https://privacyportal.onetrust.com/webform/63dc78c7-5612-4181-beae-47dead0569ee/01252c81-82df-4db5-9cb8-bf23a62a9a55'> privacy web form</a>, send an email to mailto:privacy@chick-fil-a.com, or call 1-866-232-2040.",
```

Why don't we add the email property too.
```suggestion
        "email": "privacy@chick-fil-a.com",
        "notes": "Fill out the provided <a href='https://privacyportal.onetrust.com/webform/63dc78c7-5612-4181-beae-47dead0569ee/01252c81-82df-4db5-9cb8-bf23a62a9a55'> privacy web form</a>, send an email to mailto:privacy@chick-fil-a.com, or call 1-866-232-2040.",
```

---

@WebSnke I considered that, but wasn't sure based on the contributing guidelines: 

> `email`: (optional) If you have to send an email to a company to cancel your account, add the email address here. We'll do the rest.

Email is not required but is an option for this company, which is why I included it in the `notes` but not the `email` field. Should I still be including it in the field anyways?

Also, is Markdown linking supported like the jgdm project is? Should I be formatting links in Markdown instead?

---

> Email is not required but is an option for this company, which is why I included it in the `notes` but not the `email` field. Should I still be including it in the field anyways?

Hmm, I think that is up to @tupaschoal to decide. Fortunately, he always responds quickly.

> Also, is Markdown linking supported like the jgdm project is? Should I be formatting links in Markdown instead?

Since the value of the JSON properties in injected into HTML code, you should not use Markdown styling.

---

```suggestion
        "notes": "Fill out the provided privacy web form on the link, send them an email or call 1-866-232-2040.",
        "email": "privacy@chick-fil-a.com",
```

I think that's best to avoid duplicate information.